### ISSUE TITLE
Conflit with Murdermystery he breack it :x

### ISSUE THREAD

**Issue by zephirox58** [EbonJaeger/perworldinventory-kt#182]
**Describe the bug**
Console spam with error

**To Reproduce**
1. Do 'put PWI in your server '
2. Go ' add Murdermystery plugin '

**Expected behavior**
A fix stop stop bug inventory and stop spam console <3

**Plugin version**
- perworldinventory 2.3.1

**Server version**
paper -154

**Additional context and logs**
https://pastebin.com/NSWWXBeU

**Comment by EbonJaeger** [comment]
I don't think this is an issue with PerWorldInventory. It doesn't touch anything relating to what the code in the stacktrace (error) is using. I recommend reporting the issue to the author of the MurderMystery plugin.

**Comment by zephirox58** [comment]
Is what I did first, the creator of the plugin told me that there was a conflict with perworldinventory, hence my post here ^^

**Comment by Pemigrade** [comment]
I'm also having a similar problem with Murder Mystery. However, **no other plugin** has a conflict with PerWorldInventory. For me, the problem seems to be that Murder Mystery does not clear inventories in between games and that any gold ingots picked up turn into beds in the inventory (??? no clue where this comes from lol). I'm fairly certain that the problem is on MurderMystery's side and that the developer is just being uncooperative and refusing to add support for PWI.

I also created a ticket a couple of weeks ago about my PWI conflict and was told, "Thats seems not to be a error on our side," "issue is 100% caused by perworld plugin," and "We can't help you with mm incompatible plugins." It seems like the problem is that MurderMystery doesn't support PWI and not the other way around. I asked them to add support and "If the issue is on PerWorld Inventories' side, what should I tell them that they need to fix in order to make their plugin compatible with Murder Mystery," but I received no response.

For Ebon: can you verify that my issue with inventories not clearing and gold ingots turning into beds is with Murder Mystery being incompatible and not the other way around? Thanks for your assistance!

For zeph: You need to uninstall either PWI or MuderMystery unfortunately. You could also create a bungee to a server with MurderMystery+your other plugins on it (excluding PWI) and use that one for MurderMystery games.
